test/common/upstream/random_load_balancer_fuzz_test.cc

```diff
@@ -0,0 +1,37 @@
+#include <memory>
+
+#include "test/common/upstream/load_balancer_fuzz_base.h"
+#include "test/common/upstream/random_load_balancer_fuzz.pb.validate.h"
+#include "test/fuzz/fuzz_runner.h"
+#include "test/test_common/utility.h"
+
+namespace Envoy {
+namespace Upstream {
+
+DEFINE_PROTO_FUZZER(const test::common::upstream::RandomLoadBalancerTestCase input) {
```

Comment:
reference?

Reply:
Wasn't reference for health check, do you want me to change it there too (once I do efficiency)? I changed it here.